tools/local-snapshot-runner-app/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
```

Comment:
BTW, I'd personally put it ino packages/tools. I feel like tools/ should have tools that are mostly around repo management (build tools, keys, bundle size tools, etc.). All the other tools that are more related to runtime are there

Reply:
Noticed it mostly due to presence of package lock file :)

Reply:
+1 packages/tools is the right place for something like this, as we want it to align with our client packages